README.md

```diff
@@ -142,7 +142,42 @@ terms:
         endDate: 2020-03-14
 ```
 
-A particular note to `curricularTerm` with indicates the academic semesters afected by the specific events.
+A particular note to `curricularTerm` which indicates the academic semesters afected by the specific events.
+
+## Programme Description
+
+This format describes the programme providing the list of teachers that coordinate it, the department, number of terms, contacts, link to an external reference and its description.
+
+```yml
+id: 1
+school:
+  name: Instituto Superior Engenharia Lisboa 
+  acr: ISEL
+programme:
+  name: Licenciatura em Engenharia Informática e de Computadores
+  acronym: LEIC
+termSize: 6
+department: ADEETC
+coordination:
+  - teacher: Professor Artur Jorge Ferreira
+  - teacher: Professora Cátia Raquel Jesus Vaz
+  - teacher: Professor  Manuel de Campos Lages Garcia Simão
+  - teacher: Professor Nuno Miguel Soares Datia
+  - teacher: Professor Pedro Miguel Florindo Miguéns Matutino
+contacts: ""
```

Comment:
@xploitedd this is an email field. Maybe it can be changed to `email`?

Reply:
Yes, I think it would make it more explicit